src/components/InputUnit.vue

```diff
@@ -51,12 +52,27 @@ export default class InputUnit extends Vue {
   get displayValue() {
     let convertedValue = this.toDisplay(this.value);
 
+    if (convertedValue === null || convertedValue === undefined)
+      return this.nullable ? '' : '0';
+
     return this.precision != null
       ? convertedValue.toFixed(this.precision)
       : convertedValue.toString();
   }
 
+  emitValue(v: number | null) {
+    if (this.value !== v) {
+      this.$emit('input', v);
+    } else {
+      this.htmlElement.value = this.displayValue;
+    }
+  }
+
   onChange(input: string) {
+    if (input === '' && this.nullable) {
```

Comment:
```suggestion
    if (input.trim() === '' && this.nullable) {
```

Just to be safe. Someone might try to clear the input but accidentally leave a space.